### DISCUSSION TITLE
Would love to see this updated for react-query v4!

### DISCUSSION THREAD

**Discussion by crieggalder** [kaciakmaciak/react-query-subscription#59] [General]
Thanks for the library - appreciate you putting this out there! :)

**Comment by kaciakmaciak** [comment]
Hi Chris, sorry for late reply, I was on holidays :) Yes, definitely, we'd like to make it work with react query v4. I've created an [issue](https://github.com/kaciakmaciak/react-query-subscription/issues/60) for this and will prioritize it.
